main.py

```diff
@@ -0,0 +1,54 @@
+from pypanther import PantherLogType, PantherSeverity, get_panther_rules, register
+from pypanther.rules.aws_cloudtrail_rules.aws_cloudtrail_account_discovery import AWSCloudTrailAccountDiscovery
+from pypanther.rules.aws_cloudtrail_rules.aws_cloudtrail_stopped import AWSCloudTrailStopped
+from pypanther.rules.aws_cloudtrail_rules.aws_codebuild_made_public import AWSCloudTrailCodebuildProjectMadePublic
+from pypanther.rules.aws_cloudtrail_rules.aws_console_login_without_mfa import AWSConsoleLoginWithoutMFA
+from pypanther.rules.aws_cloudtrail_rules.aws_console_login_without_saml import AWSConsoleLoginWithoutSAML
+from pypanther.wrap import exclude, include
+
+from rules.aws_cloudtrail_rules.my_first_custom_rule import MyFirstCustomRule
+from rules.aws_cloudtrail_rules.my_first_tuned_rule import MyFirstTunedRule
+
+########################################################
+## Variables
+
+onboarded_log_types = [
+    # replace with your log types
+    PantherLogType.AWS_CloudTrail,
+]
+
+########################################################
+## Overrides
```

Comment:
I like the dividers. The comments are a great pattern and very common in base config files. What if we added small one-liners (like this or similar)?

```

```suggestion
## Overrides: Customizations to Panther built-in rule attributes
```